README.md

```diff
@@ -75,6 +75,46 @@ We provide a course (free and without ads) that teaches you about natural langua
 - Does not expect prior **[PyTorch](https://pytorch.org/)** or **[TensorFlow](https://www.tensorflow.org/)** knowledge, though some familiarity with either of those will help
 </aside>
 
+### **The 🤗 Gradio Course**
+
+We provide a course (free and without ads) that teaches you how to build interactive demos for your machine learning models using libraries from the **[Hugging Face](https://huggingface.co/)** ecosystem.
+
+👉 [click here to access the 🤗 Course](https://huggingface.co/course/chapter9/1)
+
+<aside>
+💡 This course:
+
+- Its ultimate goal is to allow ML developers to easily present their work to a wide audience including non-technical teams or customers, researchers to more easily reproduce machine learning models and behavior, end users to more easily identify and debug failure points of models, and more!
+</aside>
+
+### **The 🤗 RL Course**
+
+We provide a course (free and without ads) that teaches you about Deep Reinforcement Learning using libraries from the **[Hugging Face](https://huggingface.co/)** ecosystem.
+
+👉 [click here to access the 🤗 Course](https://github.com/huggingface/deep-rl-class)
+
+<aside>
+💡 This course:
+
+- Study Deep Reinforcement Learning in theory and practice 
+- Learn to use famous Deep RL libraries
+- Train agents in unique environments
+- Publish your trained agents in one line of code to the Hugging Face Hub, and more!
+</aside>
+
+### **The Sentence Transformers educational material**
+
+We provide several tutorials on one of the most powerful libraries for industrial and academic applications. [Sentence Transformers](https://huggingface.co/sentence-transformers) allows you to create state-of-the-art embeddings from images and text for free.
+
+<aside>
+💡 We recommend following the tutorials in this order:
+
+- Guide to start with your Sentence Transformers project ([link](https://t.co/BDTP6XoATu)).
```

Comment:
I would move this one down to be the 3rd or 4th one in this list. It's not quite a tutorial, so if someone hasn't used ST at all it would be confusing.